_beats/libbeat/docs/version.asciidoc

```diff
@@ -1,6 +1,6 @@
 :stack-version: 6.1.0
```

Comment:
I think this should be `6.0.1`?

Reply:
We have our own, that also needs to get updated: https://github.com/elastic/apm-server/blob/6.0/docs/copied-from-beats/version.asciidoc